CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+cmake_minimum_required(VERSION 3.20)
+project(ltfs,
+  VERSION 2.5.0.0
```

Comment:
Please change the version number to be able to detect followings.

1. Not an official version
2. Build with CMake

Reply:
CMake does not allow to add strings in the version part. I can add a version description or try to append it to PACKAGE_VERSION on the project, that works?

Reply:
I don't think it is a good idea because we need to maintain 3 files for version management...

Can CMake allow a negative value? If so, `2.5.0.-1` is fine to me. If not, `2.5.0.9999` is candidate, but it is not my preference because `2.5.0.9999` is grater than `2.5.0.0`. 

Another candidate is `2.4.9999.9999`, but it is not so pretty...

Reply:
> Cmake also does not allow negative version numbers. I have an idea, it's ok if I make a `.in` file for cmake to define a prelim-cmake version in a `config.h`?

How we manage this?

My idea is removing version number from the `project` function(?), sorry I don't know this `project()` description is function or syntax.... From the document, version is not a mandatory field in `project()`.

Define another string variable like LTFS_VERSION and pass the version number by `-D` option to the compiler.